views/account/account.handlebars

```diff
@@ -145,7 +145,10 @@
 				<div class="setting-card">
 					<h2 class="header">{{ locale.account.settings.settingCards.discord }}</h2>
 						{{#if discordUser}}
-					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}{{#ifCond discordUser.discriminator '#0'}}{{else}}#{{ discordUser.discriminator }}{{/ifCond}}</p>
+					<p>{{ locale.account.settings.settingCards.connectedToDiscord }} {{ discordUser.username }}
+					{{#if discordUser.discriminator}}
```

Comment:
Again, see the above `#if` checks for how to actually check if a value is matching another. This always resolves to true since the value will always exist even if it's the string `0`